blog/views.py

```diff
@@ -57,3 +57,15 @@ class DeletePostView(DeleteView):
     model = Post
     template_name = 'delete_post.html'
     success_url = reverse_lazy('home')
+
+
+class AddCommentView(CreateView):
+    model = Comment
+    form_class = AddCommentForm
+    template_name = 'add_comment.html'
+    success_url = reverse_lazy('home')
```

Comment:
Why user will go to home page after added comment to post?

Reply:
fixing this with redirection to current post page